docs/guides/gradle.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Gradle configuration
+headline: Documentation
+bodyclass: docs
+layout: docs
+---
+
+# Configuring a project with Gradle
+
+Spine provides build-time tools for code generation and analysis. These tools are implemented
+as [Gradle plugins](https://docs.gradle.org/current/userguide/plugins.html) — an extendable, uniform
+way to tap into a project build process.
+
+The minimal Gradle configuration you will need to start a new project is:
+
+<embed-code file="examples/hello/build.gradle" start="plugins" end="}"></embed-code>
+```groovy
+plugins {
+    id 'io.spine.tools.gradle.bootstrap' version '1.5.24'
+}
+```
+
+Place the config into your root `build.gradle` file and execute a Gradle build. This will apply
+the Spine Bootstrap plugin to your project.
+
+You can also find this declaration on the [Gradle Plugin Portal](https://plugins.gradle.org/plugin/io.spine.tools.gradle.bootstrap),
+or on our [Getting Started page]({{site.baseurl}}/docs/quick-start).
+
+## Spine Bootstrap plugin 
+
+Spine Bootstrap plugin (Bootstrap for short) can configure a Spine Java server and JS client for
+you.
+
+The idea is to use Gradle subprojects for different parts of the system.
```

Comment:
Let's connect this sentence to the previous one and the content which follows after. 

Something like this:

```
We recommend having separate Gradle subprojects for domain model definition, server 
implementation, and client code. Bootstrap is applied a bit differently in each
of these cases.
```

Also, the previous sentence says nothing about the model. Maybe we could rewrite it to give a more generic definition for the plugin per se:

```
Spine Bootstrap plugin (Bootstrap for short) serves to automate the configuration of the modules
in your Spine-based app.
```